athena-hbase/src/main/java/com/amazonaws/athena/connectors/hbase/HbaseConnectionFactory.java

```diff
@@ -125,7 +125,7 @@ private Connection createConnection(String host, String masterPort, String zooke
                 config.set(nextConfig.getKey(), nextConfig.getValue());
             }
 
-            Map<String, String> configOptions = System.getenv();
+            Map<String, String> configOptions = new HbaseEnvironmentProperties().createEnvironment();
```

Comment:
Move this to constructor. we don't want to call glue each time. This should only do it at start up time

Reply:
#non-blocker; created internal ticket.